### PR TITLE
rel: prepare changelog for release

### PR DESCRIPTION
**Pull request by codeboten**
Ran `make update-changelog` as per the [release doc](https://github.com/honeycombio/terraform-aws-integrations/blob/main/RELEASING.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,28 @@
-<a name="v1.3.0"></a>
-# [v1.3.0 🌈](https://github.com/honeycombio/terraform-aws-integrations/releases/tag/v1.3.0) - 18 Apr 2024
+<a id="v1.3.1"></a>
+# [v1.3.1 🌈](https://github.com/honeycombio/terraform-aws-integrations/releases/tag/v1.3.1) - 2024-04-24
```
Comment (VinozzZ) [inline CHANGELOG.md:2]: the date needs to be updated?

**Review by JamieDanielson — COMMENTED** [review]
this is a newer project for our team and doesn't follow our usual release flow. Can you write some notes in the PR description for what you did? like was this output a result of make update-changelog from [releasing doc](https://github.com/honeycombio/terraform-aws-integrations/blob/main/RELEASING.md)?

File: CHANGELOG.md (hunk 1 above)
Comment (codeboten) [reply CHANGELOG.md:2]: the version of `changelog-from-release` https://github.com/rhysd/changelog-from-release/releases/tag/v3.8.0 updates the date to be the standard ISO format

Comment (JamieDanielson) [reply CHANGELOG.md:2]: Ooh I think it auto-generated the date based on the date of the draft release. The draft release was created on April 24. If we change that date to today then re-running this should use today's date.

Comment (JamieDanielson) [reply CHANGELOG.md:2]: Oh I don't know if it's possible. Weird. Maybe delete that release and create a new one?

**Review by JamieDanielson — COMMENTED** [review]
Also since this is preparing a release, we would usually use rel: as the title prefix

File: CHANGELOG.md (hunk 1 above)
Comment (codeboten) [reply CHANGELOG.md]: fixed now

**Review by JamieDanielson — APPROVED** [review]
let it go

**Comment by codeboten** [comment]
Removing the apply project labels workflow as part of this, the workflow has not worked in 6 months in this repo

**Pull request merged**